hazelcast-client/src/test/java/com/hazelcast/client/queue/ClientQueueDisruptionTest.java

```diff
@@ -54,25 +57,51 @@ public void clientsConsume_withNodeShutdown() throws InterruptedException {
             cluster.getRandomNode().getQueue("Q2").offer(i);
         }
 
-        int expect = 0;
+        int expectCount = 0;
         for (int i = initial; i < max; i++) {
 
             if (i == max / 2) {
                 cluster.shutdownRandomNode();
             }
-
-            assertTrue(cluster.getRandomNode().getQueue("Q1").offer(i));
-            assertTrue(cluster.getRandomNode().getQueue("Q2").offer(i));
-
-            TestCase.assertEquals(expect, client1.getQueue("Q1").poll());
-            TestCase.assertEquals(expect, client2.getQueue("Q2").poll());
-
-            expect++;
+            final int index = i;
+
+            makeSureExactlyOneSuccessfulRun(new AssertTask() {
```

Comment:
Why not call it assertExactlyOneSuccessfulRun.

Assert is the prefix we use to assert things.